bottlecap/src/logs/flusher.rs

```diff
@@ -1,53 +1,69 @@
 use crate::logs::aggregator::Aggregator;
+use std::collections::HashMap;
 use std::sync::{Arc, Mutex};
 use tokio::task::JoinSet;
 use tracing::{debug, error};
 
 pub struct Flusher {
     api_key: String,
-    site: String,
+    fqdn_site: String,
     client: reqwest::Client,
     aggregator: Arc<Mutex<Aggregator>>,
 }
 
+#[must_use]
+pub fn build_fqdn_logs(site: String) -> String {
+    format!("https://http-intake.logs.{site}")
+}
+
 #[allow(clippy::await_holding_lock)]
 impl Flusher {
     pub fn new(api_key: String, aggregator: Arc<Mutex<Aggregator>>, site: String) -> Self {
         let client = reqwest::Client::new();
         Flusher {
             api_key,
-            site,
+            fqdn_site: site,
             client,
             aggregator,
         }
     }
-    pub async fn flush(&self) {
+    pub async fn flush(&self) -> HashMap<String, Result<(), String>> {
```

Comment:
I don't think we should return anything from this method, especially not a hashmap for results? We're doing nothing with them.

Reply:
It is just to verify that the request was executed correctly. Right now it is used only on tests, but I guess it will be needed when we do the retry